prepare-image.sh

```diff
@@ -11,16 +11,16 @@ echo "tsflags=nodocs" >> /etc/dnf/dnf.conf
 dnf install -y python3 python3-requests 'dnf-command(config-manager)'
 
 # RPM install #
-if [[ $INSTALL_TYPE == "rpm" ]]; then
+if [[ "${INSTALL_TYPE}" == "rpm" ]]; then
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/plugins/module_utils/tripleo_repos/main.py | python3 - -b master current-tripleo
     # NOTE(elfosardo): enable CRB repo for more python3 dependencies
     dnf config-manager --set-enabled crb
     dnf upgrade -y
-    xargs -rtd'\n' dnf install -y < $IRONIC_PKG_LIST
+    xargs -rtd'\n' dnf install -y < "${IRONIC_PKG_LIST}"
```

Comment:
same here, probably better add disable=SC2086

Reply:
IRONIC_PKG_LIST is a file name and its read by `<`, so here quotes are correct.